autogluon/scheduler/rl_scheduler.py

```diff
@@ -70,7 +70,7 @@ class RLScheduler(FIFOScheduler):
     >>> scheduler.join_jobs()
     >>> scheduler.get_training_curves(plot=True)
     """
-    def __init__(self, train_fn, args=None, resource=None, checkpoint='./exp/checkpoint.ag',
+    def __init__(self, train_fn, args=None, resource=None, searcher=None, checkpoint='./exp/checkpoint.ag',
```

Comment:
can you also add:

```
if not isinstance(searcher, RLSearcher):
```
before this line https://github.com/awslabs/autogluon/blob/2ffc14ab69aa1d3224a36347b0373ee062684173/autogluon/scheduler/rl_scheduler.py#L84

Reply:
have added. Thanks